sources/packages/backend/apps/api/src/services/student/student.service.ts

```diff
@@ -194,6 +196,17 @@ export class StudentService extends RecordDataModelService<Student> {
       throw error;
     }
 
+    // If sfasIndividual wasn't found, check for a partial match
+    if (!sfasIndividual) {
```

Comment:
Not a blocker but it can also be the `else` condition from line 183.
```ts
if (sfasIndividual) {
    student.disabilityStatus = this.getDisabilityStatus(
      sfasIndividual.pdStatus,
      sfasIndividual.ppdStatus,
    );
    student.disabilityStatusEffectiveDate = this.getDisabilityEffectiveDate(
      sfasIndividual.ppdStatusDate,
      student.disabilityStatus,
    );
  } else {
    // If sfasIndividual wasn't found, check for a partial match.
    await this.getStudentPartialMatch(
      user,
      student,
      studentSIN,
      auditUserId,
      externalEntityManager,
    );
  }
```

Reply:
updated